public/app/plugins/datasource/prometheus/configuration/ConfigEditor.tsx

```diff
@@ -18,7 +18,11 @@ export const ConfigEditor = (props: Props) => {
         sigV4AuthToggleEnabled={config.sigV4AuthEnabled}
       />
 
-      <AlertingSettings<PromOptions> options={options} onOptionsChange={onOptionsChange} />
+      <AlertingSettings<PromOptions>
```

Comment:
Shouldn't this be visible only if `ngalert` is enabled?

Reply:
oops, good point

Reply:
fixed